.tools/readme.py

```diff
@@ -0,0 +1,20 @@
+import re
```

Comment:
```suggestion
"""Pre-commit hook to check and update README.md."""

import re
```

Reply:
How about
"""Script to update README.md used by pre-commit."

Reply:
I took the liberty to update and apply the suggestion. Feel welcome to tweak further. :)